app/celer-sim/simple-driver.py

```diff
@@ -47,6 +47,9 @@ def strtobool(text):
     'msc': "urban" if core_geo == "vecgeom" else "none",
     'eloss_fluctuation': True,
     'lpm': True,
+    'cerenkov': True,
+    'scintillation': True,
+    'optical_rayleigh': True,
```

Comment:
Oh good catch, since GeantPhyscisOptions was reverted after our discussion, these should have been too. They were originally just set to true due to our initial planning: if the data is not available from the gdml, the optical physics will not be initialized. So it would work for both simple-cms and for an optical test, such as the lar-sphere.